lib/pinch_hitter/message/message_store.rb

```diff
@@ -16,11 +11,11 @@ def initialize(message_directory)
 
     def load(file, overrides={})
       filename = find_filename file
-      if filename =~ /xml$/
-        xml_message filename, overrides
-      else
-        json_message filename, overrides
-      end
+      file_extension = filename[/(?<=\.)[^.]+$/]
+      fail "Filename must have an extension to indicate type of response (#{filename})" if file_extension.nil?
```

Comment:
Hmm...this would potentially break existing test suites.  We'll definitely want a good Changelog message for this.

Reply:
Yeah, I figured that would be the case. I documented the breaking changes in [this commit message](https://github.com/stevenjackson/pinch_hitter/pull/13/commits/18a8fb6d463bf5403c22af733f6c3bc7a6707a31), but I think the only user-facing breakages are the last two bullets:

  - If the content type of a request is determined not to be JSON or XML, it defaults to Plain Text, instead of XML
  - Files inside the message store directory must be named with their appropriate file extension (.xml, .json, or .txt for plain text)